app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,28 @@
 <resources>
     <string name="app_name">login</string>
+
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="welcome">Welcome</string>
+    <string name="login">Login…</string>
+    <string name="password">Password…</string>
+    <string name="log_in">Log In</string>
+    <string name="hello">Hello</string>
+    <string name="log_out">Log Out</string>
+    <string name="app_name_splash">App Name</string>
+
+    <string name="error">Ошибка!</string>
+
+    <string name="login_successful">Вход выполнен</string>
```

Comment:
localisation should be in properly xml file

Reply:
done